plugins/modules/cloud/smartos/vmadm.py

```diff
@@ -430,10 +430,8 @@ def get_vm_prop(module, uuid, prop):
             msg='Invalid JSON returned by vmadm for uuid lookup of {0}'.format(prop),
             details=to_native(e), exception=traceback.format_exc())
 
-    if len(stdout_json) > 0 and prop in stdout_json[0]:
-        return stdout_json[0][prop]
-    else:
-        return None
+    if stdout_json > 0:
```

Comment:
I don't think that `> 0` will work for a dict.
```suggestion
    if stdout_json:
```